pkgs/applications/misc/nerd-font-patcher/default.nix

```diff
@@ -1,36 +1,38 @@
-{ python3Packages, lib, fetchFromGitHub }:
+{ python3Packages, lib, unzip, fetchFromGitHub }:
 
 python3Packages.buildPythonApplication rec {
   pname = "nerd-font-patcher";
-  version = "2.2.2";
+  version = "3.0.0";
 
   # This uses a sparse checkout because the repo is >2GB without it
   src = fetchFromGitHub {
     owner = "ryanoasis";
     repo = "nerd-fonts";
     rev = "v${version}";
     sparseCheckout = [
-      "font-patcher"
-      "/src/glyphs"
+      "FontPatcher.zip"
     ];
-    sha256 = "sha256-boZUd1PM8puc9BTgOwCJpkfk6VMdXLsIyp+fQmW/ZqI=";
+    sha256 = "sha256-7pI2t9SHv8y8pEVHqbIECt2I9HGdgEKwWFLM3/DCK8g=";
   };
 
   propagatedBuildInputs = with python3Packages; [ fontforge ];
 
   format = "other";
+  nativeBuildInputs = [ unzip ];
 
   postPatch = ''
-    sed -i font-patcher \
+    unzip FontPatcher.zip -d nerdpatcher
+    sed -i nerdpatcher/font-patcher \
       -e 's,__dir__ + "/src,"'$out'/share/nerd-font-patcher,'
   '';
 
   dontBuild = true;
 
   installPhase = ''
-    mkdir -p $out/bin $out/share/nerd-font-patcher
-    install -Dm755 font-patcher $out/bin/nerd-font-patcher
-    cp -ra src/glyphs $out/share/nerd-font-patcher
+    mkdir -p $out/bin $out/share/nerd-font-patcher $out/bin
+    install -Dm755 nerdpatcher/font-patcher $out/bin/nerd-font-patcher
+    cp -ra nerdpatcher/src/glyphs $out/share/nerd-font-patcher
+    cp -ra nerdpatcher/bin $out/bin/bin
```

Comment:
the bin directory contains a script directory with Python classes. Please copy them to a lib directory.